code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -103,8 +113,7 @@
 		holder_type = null
 
 	species = all_species[new_species]
-	if(species.language)
-		add_language(species.default_language)
+	add_language(species.default_language)
```

Comment:
This is a specific override for a diona nymph, which will only ever be diona species.

If species or species.default_language are either null or some strange value, this call will throw a quite descriptive runtime error alerting to that fact.  Its something that should never happen, but we want to know about it if it somehow does, so in that case the runtime error is desired and a safety check is not.